src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/GeoJsonOutputFormatWfsTest.java

```diff
@@ -91,8 +91,19 @@ public void testGetGeoJsonResponseWfs20() throws Exception {
                 getAsJSON(
                         "wfs?request=GetFeature&version=2.0"
                                 + "&typenames=st_gml32:Station_gml32&outputFormat=application/json");
+        print(response);
```

Comment:
A leftover?

Reply:
Yes, a leftover indeed. I would not want you to get bored :-p